benchmarks/Exporter/NonExporterBenchmarks.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+using BenchmarkDotNet.Attributes;
+
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+using OpenTelemetry.Trace.Configuration;
+using OpenTelemetry.Trace.Export;
+
+namespace Benchmarks.Exporter
+{
+    /// <summary>
+    /// These benchmarks compare OpenTelemetry without an Exporter vs with a NoOp Exporter
+    /// </summary>
+    /// <remarks>
+    /// To run: .\Benchmarks.exe --filter NonExporterBenchmarks
+    /// </remarks>
+    [MemoryDiagnoser]
+#if !NET462
+    [ThreadingDiagnoser]
+#endif
+    public class NonExporterBenchmarks
+    {
+
+        [Params(1, 10)]
+        public int NumberOfTraces { get; set; }
+
+        [Params(1, 10)]
+        public int NumberOfSpans { get; set; }
+
+
+        [Benchmark]
+        public void NoExporter()
```

Comment:
No exporter means internally it uses NoOpSpanProcessor. Comparing that with NoOpExporter is not very useful.

Not an expert here, but something more useful would be to compare NoOpExporter vs JagerExporter vs ApplicationInsightsExporter.

Or NoOpExporter with AlwaysOnSampler, vs NoOpExporter with AlwaysOffSampler